Sources/ArcGISToolkit/Components/FormView/FormInputs/ComboBoxInput.swift

```diff
@@ -75,9 +78,10 @@ struct ComboBoxInput: View {
         self.noValueLabel = input.noValueLabel
         self.noValueOption = input.noValueOption
         
-        _inputModel = StateObject(
-            wrappedValue: FormInputModel(fieldFormElement: element)
-        )
+        value = element.value
+        formattedValue = element.formattedValue
+        isRequired = element.isRequired
+        isEditable = element.isEditable
```

Comment:
Setting state properties in the initializer has the same problem as setting state objects in the initializer—if the values passed to the view's initializer change while the view is on screen, the state properties won't be updated, leaving the view in an inconsistent state. Instead, the state properties will get updated automatically as soon as observation starts since the first value of the streams is the current value. That also means that setting `selectedValue` in `onAppear` is unnecessary. Without that, the formatted value is only used from inside the action closure of `onChangeOfValue(of:action:)`, so the `formattedValue` state property could go away.